website/www/site/data/en/powered-by.yaml

```
@@ -0,0 +1,22 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- title: Commercial projects
+  body: Learn how the companies use Apache Beam and make their work easier.
+  icon: /icons/commercial-icon.svg
+  link: /powered-by/commercial
+- title: Open source projects
+  body: Learn how the companies use Apache Beam and make their work easier.
```

There should be some different copy for OS projects. Do we need to get some from @griscz?

---

Yes, I think so, there isn't any in TDD for these cards

---

Propably yes

---

I think Gris is OOO, lets go with this:
```suggestion
  body: Find other Open Source projects that use Apache Beam.
```